red4ext-sys/src/interop.rs

```diff
@@ -164,6 +162,24 @@ impl TweakDbId {
             length: str.len() as u8 + base.length,
         }
     }
+
+    pub const fn as_u64(&self) -> u64 {
```

Comment:
this should be called `to_u64` according to the Rust API guidelines: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Reply:
should probably look like this because `self` is `Copy`
```suggestion
    pub const fn to_u64(self) -> u64 {
```